sample-compose/domain/src/main/java/co/nimblehq/sample/compose/domain/usecase/IsFirstTimeLaunchPreferencesUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.sample.compose.domain.usecase
+
+import co.nimblehq.sample.compose.domain.repository.AppPreferencesRepository
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+class IsFirstTimeLaunchPreferencesUseCase @Inject constructor(
```

Comment:
However, we can keep the `Preferences` postfix here to explicit we will get it from `Preferences` 🤓